jetty-openid/src/test/java/org/eclipse/jetty/security/openid/OpenIdProvider.java

```diff
@@ -289,7 +299,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
             long expiry = System.currentTimeMillis() + Duration.ofMinutes(10).toMillis();
```

Comment:
Avoid `System.currentTimeMillis()`, so:

```suggestion
            Instant expiry = Instant.now().plusSeconds(10);
```